cartoframes/viz/html/html_layout.py

```diff
@@ -64,6 +64,7 @@ def _parse_html_content(self, maps, size, show_info=None, theme=None, _carto_vl_
             is_embed=is_embed,
             is_static=is_static,
             map_height=map_height,
+            layer_selector=False,
```

Comment:
Why is this False?

Reply:
I fixed it here https://github.com/CartoDB/cartoframes/pull/1598